wp-content/themes/orient-theme/taxonomy-series.php

```diff
@@ -1,18 +1,35 @@
 <?php
 
-get_header(); ?>
+get_header();
 
+//create a copy of wp_query
```

Comment:
You've done a great job commenting your code in this PR -- I'm glad that that seems instinctual to you. However, some code comments are better than others, and I tend to shy away from comments that explain a single line of readable code.

It doesn't seem like the following line (`$copy = clone $wp_query;`) is difficult enough to understand that it needs a comment explaining it, so I might suggest getting rid of this comment.